Site/bc_weather_station_dashboard/website/templates/home.html

```diff
@@ -92,15 +91,42 @@ <h2>
 
                 <!-- Row 1 -->
                 <div class="row pt-4 justify-content-center">
+
                     <!-- Temperature Card -->
-                    <div class="col-md-4">
-                        <div class="card">
-                            <div class="card-body text-center">
-                                <h5 class="card-title">Temperature</h5>
-                                <h2 class="card-text">34 C</h2>
+                    <div class="card rounded shadow bg-info text-white mb-md-0 mb-4">
+                        <div class="card-body text-center p-4">
+                            <!-- Temp icon and title -->
+                            <h5 class="card-title mb-2 d-flex align-items-center justify-content-center">
+                                <span class="mr-2">
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="white"
+                                        class="bi bi-thermometer" viewBox="0 0 16 16">
+                                        <path d="M8 14a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3" />
+                                        <path
+                                            d="M8 0a2.5 2.5 0 0 0-2.5 2.5v7.55a3.5 3.5 0 1 0 5 0V2.5A2.5 2.5 0 0 0 8 0M6.5 2.5a1.5 1.5 0 1 1 3 0v7.987l.167.15a2.5 2.5 0 1 1-3.333 0l.166-.15z" />
+                                    </svg>
```

Comment:
This should be an svg in the static folder, not an inline tag.

Reply:
what file should it go in?

Reply:
nvm

Reply:
look how the calendar svg is saved. It should be its own file
![image](https://github.com/carsondrobe/Fellas/assets/31668119/e6a36fa3-ac9e-4e5d-b2a1-e8ed08460124)

Reply:
yeah I made a folder for images in static and put it there 👍🏻